src/components/MainPage.vue

```diff
@@ -14,13 +14,17 @@
 
       <v-spacer />
 
-      <v-btn
-        text
-        @click="() => openExternal('https://github.com/baruchiro/israeli-bank-scrapers-desktop/issues/new')"
-      >
-        <span class="mr-2">Report a problem</span>
-        <v-icon>mdi-chat-alert-outline</v-icon>
-      </v-btn>
+      <report-problem-dialog>
+        <template v-slot="{ on }">
```

Comment:
I need to refactor this to pass the `activator` with `ref` and not with `template`